pkg/controllers/disruption/controller.go

```diff
@@ -69,6 +69,13 @@ func NewController(clk clock.Clock, kubeClient client.Client, provisioner *provi
 	cp cloudprovider.CloudProvider, recorder events.Recorder, cluster *state.Cluster, queue *orchestration.Queue,
 ) *Controller {
 	c := MakeConsolidation(clk, cluster, kubeClient, provisioner, cp, recorder, queue)
+
+	// Generate eventually disruptable reason based on a combination of drift and cloudprovider disruption reason
+	eventualDisruptionMethods := []Method{}
+	for _, reason := range append([]v1.DisruptionReason{v1.DisruptionReasonDrifted}, cp.DisruptionReasons()...) {
```

Comment:
nit: `lo.Map()`?

Reply:
How does `lo.Map()` improve things here?

Reply:
It's a nit, you could use `lo.Map` here if you want

Reply:
I'm gonna leave this as it, I'm fully groking how this would work with `lo.Map()`